### DISCUSSION TITLE
Organization name taken

### DISCUSSION THREAD

**Discussion by factiiv** [community/community#50941] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, 
Was trying to link our organization name to my personal account.  
The user name that i am using is the same as the organization but when creating an organization it says the name is taken.  

is there something i am doing wrong.  If the personal account is the same as the organization name does it show that the organization name is taken ?  

Thanks

**Comment by meppu** [comment]
>  If the personal account is the same as the organization name does it show that the organization name is taken?

Yes, you can't claim same name because of that.

You can directly contact with github support to help you, or if you have time, you can delete your account and your username will be available to anyone for use after 90 days.

Second option is not a really good idea so you should contact with them from [here](https://support.github.com/contact/account). github no longer accepts requests to release dormant usernames but in this case you want to move your name to an organization so it can be accepted.

You can also create an organization with similar name. For example "factiiv-org"

**Comment by factiiv** [comment]
ok,
So because i have the personal user name as factiiv, I cannot create an
organizatoin name factiiv

Thanks

Bob
